Assets/MixedRealityToolkit.Providers/Oculus/GenericOculusAndroidController.cs

```diff
@@ -0,0 +1,202 @@
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(3, "Trigger Touch", AxisType.Digital, DeviceInputType.TriggerTouch, ControllerMappingLibrary.AXIS_9),
+            // HTC Vive Controller - Left Controller Grip Button (8)
+            // Oculus Touch Controller - Axis1D.PrimaryHandTrigger
+            // Valve Knuckles Controller - Left Controller Grip Average
+            // Windows Mixed Reality Controller - Left Grip Button Press
+            new MixedRealityInteractionMapping(4, "Grip Trigger Position", AxisType.SingleAxis, DeviceInputType.Trigger, ControllerMappingLibrary.AXIS_11),
+            // HTC Vive Controller - Left Controller Trackpad (2)
+            // Oculus Touch Controller - Axis2D.PrimaryThumbstick
+            // Valve Knuckles Controller - Left Controller Trackpad
+            // Windows Mixed Reality Controller - Left Thumbstick Position
+            new MixedRealityInteractionMapping(5, "Trackpad-Thumbstick Position", AxisType.DualAxis, DeviceInputType.Touchpad, ControllerMappingLibrary.AXIS_1, ControllerMappingLibrary.AXIS_2, false, true),
+            // HTC Vive Controller - Left Controller Trackpad (2)
+            // Oculus Touch Controller - Button.PrimaryThumbstick
+            // Valve Knuckles Controller - Left Controller Trackpad
+            // Windows Mixed Reality Controller - Left Touchpad Touch
+            new MixedRealityInteractionMapping(6, "Trackpad-Thumbstick Touch", AxisType.Digital, DeviceInputType.TouchpadTouch, KeyCode.JoystickButton16),
+            // HTC Vive Controller - Left Controller Trackpad (2)
+            // Oculus Touch Controller - Button.PrimaryThumbstick
+            // Valve Knuckles Controller - Left Controller Trackpad
+            // Windows Mixed Reality Controller - Left Thumbstick Press
+            new MixedRealityInteractionMapping(7, "Trackpad-Thumbstick Press", AxisType.Digital, DeviceInputType.TouchpadPress, KeyCode.JoystickButton8),
+            // HTC Vive Controller - Left Controller Menu Button (1)
+            // Oculus Touch Controller - Button.Three Press
+            // Valve Knuckles Controller - Left Controller Inner Face Button
+            // Windows Mixed Reality Controller - Left Menu Button
+            new MixedRealityInteractionMapping(8, "Unity Button Id 2", AxisType.Digital, DeviceInputType.ButtonPress, KeyCode.JoystickButton2),
+            // Oculus Touch Controller - Button.Four Press
+            // Valve Knuckles Controller - Left Controller Outer Face Button
+            new MixedRealityInteractionMapping(9, "Unity Button Id 3", AxisType.Digital, DeviceInputType.ButtonPress, KeyCode.JoystickButton3),
+            new MixedRealityInteractionMapping(10, "WMR Touchpad Touch", AxisType.Digital, DeviceInputType.TouchpadTouch, KeyCode.JoystickButton18),
```

Comment:
Not sure if WMR is appropreate here.

Reply:
This class  could very well be empty. I just copied another over so there was a generic version for any fallback to come. Since there is no fallback for Oculus GO I was not sure what a fallback could look like.

Reply:
Well, you should pre-fil it with appropriate defaults, the consumer has the full access to change this for their project.  But it should be clean and not retain any "copy" config / functions that are not specific to this device.